### ISSUE TITLE
Please don' use the same version number for different release binaries under the same name

### ISSUE THREAD

**Issue by rwbc** [Orbital-Web/Raphael#3]
This will make testing very difficult. Meanwhile you had at least four different 1.7 versions silently updated in the original release
section and it will be hard to follow what was tested and had this or that bug.

You could also add the commit hash or a date to the version variable, if you don't want to make a different incremented release for now.

Guenther (RWBC)

P.S. All your compilations so far are non-statical, which means the user has to add missing gcc lib dlls himself (better compile with the -static flag) and your newest 1.7 release binary misses one more dll, `openal32.dll`, while the binary is now four times the size as before?
(Edit: seems you mixed up the GUI with the uci engine release with the last one?)

**Comment by Orbital-Web** [comment]
Hi, thanks for letting me know. I'll make sure to separate the releases or add patch versioning to the files from now on. It was mostly a refactoring of the code and no changes were made to the engine itself.

The missing openal32.dll has been an issue for a while and I haven't quite figured out the cause, though I will take a deeper look at it. I will also try to figure out how to compile SFML statically on Ubuntu. I'm also not too sure why the binaries grew in size, and I also don't know what you mean by the GUI and UCI engine being mixed up.

**Comment by Orbital-Web** [comment]
Hi, quick update. Seems like statically linking openal32 won't be possible due to licensing issues. I will add instructions under Getting Started to make it more clear.

**Comment by rwbc** [comment]
> Hi, thanks for letting me know. I'll make sure to separate the releases or add patch versioning to the files from now on. It was mostly a refactoring of the code and no changes were made to the engine itself.
> 
> The missing openal32.dll has been an issue for a while and I haven't quite figured out the cause, though I will take a deeper look at it. I will also try to figure out how to compile SFML statically on Ubuntu. I'm also not too sure why the binaries grew in size, and I also don't know what you mean by the GUI and UCI engine being mixed up.

Well, I suspected you might have mixed up the GUI (main) and UCI engine (Raphael) in the last win binary release, because just a few hourse before (you exchanged the binaries at least two times...), the size of the plain engine w/o GUI was much smaller and did not need the openal32.dll.
It seems it is not the case. Yet I wonder why the plain uci engine should need an audio library at all?

**Comment by rwbc** [comment]
And BTW in win, versions 1.7 display unix time stamps in the pv output. This was not the case with version 1.6.

```
uci
id name Raphael
id author Rei Meguro
uciok
isready
position startpos
go infinite
info depth 1 nodes 41 score cp 43 pv d2d4
info depth 2 nodes 188 score cp 0 pv d2d4 d7d5
info depth 3 nodes 759 score cp 37 pv d2d4 d7d5 g1f3
info depth 4 nodes 3903 score cp 6 pv e2e4 d7d5 f1b5 c8d7
info depth 5 nodes 15479 score cp 34 pv b1c3 d7d5 d2d4 g8f6 g1f3
info depth 6 nodes 59038 score cp 0 pv b1c3 d7d5 d2d4 g8f6 g1f3 b8c6
info depth 7 nodes 152891 score cp 36 pv d2d4 d7d5 g1f3 g8f6 b1c3 b8c6 c1f4
info depth 8 nodes 656100 score cp 13 pv e2e4 b8c6 g1f3 e7e5 b1c3 g8f6 d2d4 f8d6
info depth 9 nodes 1889379 score cp 26 pv e2e4 d7d5 e4d5 d8d5 b1c3 d5e6 f1e2 g8f6 d2d4
```
latest 1.7 (the one which needs the extra dll, but two other '1.7' versions from before behave the same)
```
uci
id name Raphael 1.7
id author Rei Meguro
option name Hash type spin default 192 min 1 max 2560
uciok
isready
readyok
position startpos
go infinite
info depth 1 time 1734338352825 nodes 41 score cp 43 nps 0 pv d2d4
info depth 2 time 1734338352826 nodes 186 score cp 0 nps 0 pv d2d4 d7d5
info depth 3 time 1734338352827 nodes 735 score cp 37 nps 0 pv d2d4 d7d5 g1f3
info depth 4 time 1734338352833 nodes 3529 score cp 6 nps 0 pv e2e4 d7d5 f1b5 c8d7
info depth 5 time 1734338352850 nodes 12505 score cp 34 nps 0 pv b1c3 d7d5 d2d4 g8f6 g1f3
info depth 6 time 1734338352893 nodes 35726 score cp 0 nps 0 pv b1c3 d7d5 d2d4 g8f6 g1f3 b8c6
info depth 7 time 1734338353022 nodes 106323 score cp 36 nps 0 pv d2d4 d7d5 g1f3 g8f6 b1c3 b8c6 c1f4
info depth 8 time 1734338353458 nodes 349456 score cp 13 nps 0 pv e2e4 b8c6 g1f3 e7e5 b1c3 g8f6 d2d4 f8d6
info depth 9 time 1734338355069 nodes 1263050 score cp 24 nps 0 pv e2e4 e7e6 b1c3 d7d5 e4d5 e6d5 d2d4 g8f6 g1f3
```

**Comment by Orbital-Web** [comment]
I really appreciate your help in debugging. Turns out there's still quite a lot of careless mistakes in my code (maybe I should start adding test cases...).
Anyways, the uci engine needing openal is fixed, as I was compiling with a bunch of unnecessary object files. I also cleared up the README to help with that. 
I also fixed the issue with the timestamp. Turns out I wasn't resetting the start_time when the argument was infinite. It's a bit weird that it was working in 1.6, unless I accidentally deleted that line or something..

I will push the code in the next hour or so and add the new releases with a different version number to make things clear.

**Comment by Orbital-Web** [comment]
There's also a few changes I can make such as making the transposition table entry fit in 16 instead of 24 bytes, and compiling with the O3 flag which makes a significant difference so I might make a new release entirely.

**Comment by Orbital-Web** [comment]
I've pushed the changes as part of a [new release](https://github.com/Orbital-Web/Raphael/releases/tag/v1.7.6). Please let me know if you notice anything weird. Thanks a lot!

**Comment by SzotsGabor** [comment]
Again, version is 1.7.6.2 but 1.7 is displayed.

**Comment by Orbital-Web** [comment]
This has been fixed with the latest release
